docs/snark-work.md

```diff
@@ -115,35 +109,39 @@ After this, the updated SNARK pool with the completed (but not yet included in a
 ![image](https://github.com/JanSlobodnik/pre-publishing/assets/60480123/f02fc1f4-e30e-4296-9a20-b7b57e2cf4a1)
 
 
-
-
-![image](https://github.com/JanSlobodnik/pre-publishing/assets/60480123/e9067fcc-2180-46b3-b445-a03ac77c2b16)
+<img width="318" alt="image" src="https://github.com/openmina/openmina/assets/60480123/46ec6804-b767-4054-aaeb-0287ab9cda09">
 
 
 ### Receiving SNARK from Rust node
 
+Rust node sends SNARK via P2P.
 
+<img width="342" alt="image" src="https://github.com/openmina/openmina/assets/60480123/067de8a5-246e-4b59-a85b-7f2393cc19c3">
```

Comment:
@JanSlobodnik was the switch from markdown image tags to these html tags (with fixed width) intentional?